src/routes/Book.svelte

```diff
@@ -25,15 +27,78 @@
 		margin: 20px;
 		border: solid 5px navy;
 		border-radius: 15px;
-	}
-
-	p {
-		color: white;
+	} */
+	h1 {
+    color: #73363D;
+  }
+	.book-info {
+		color: #73363D;
 		font-family: sans-serif;
 		font-size: 1.5em;
+		display: flex;
+		flex-direction: column;
+		margin-right: 10px;
+    margin-left: 10px;
+	}
+	button {
+    height: 50px;
+    width: 200px;
+    background-color: #73363D;
+    border-radius: 5px;
+    color: white;
+    font-size: 20px;
+    margin-right: 10px;
+    margin-left: 10px;
+  }
```

Comment:
can add margin: 20px for all around spacing
Can also add a longer width to make bigger for longer button titles